extensions/kafka/src/main/java/io/deephaven/kafka/ingest/KafkaStreamConsumer.java

```diff
@@ -13,5 +13,12 @@
  * Consumer for lists of ConsumerRecords coming from Kafka. The StreamFailureConsumer is extended so that we can report
  * errors emanating from our consumer thread.
  */
-public interface KafkaStreamConsumer extends Consumer<List<? extends ConsumerRecord<?, ?>>>, StreamFailureConsumer {
+public interface KafkaStreamConsumer extends StreamFailureConsumer {
+    /**
+     * Consume a list of ConsumerRecords coming from Kafka.
+     *
+     * @param records the records to consume
+     * @return the total number of message bytes processed
+     */
+    long consume(List<? extends ConsumerRecord<?, ?>> records);
```

Comment:
I don't have strong opinions - but do you prefer `consume` vs `accept`?

Reply:
I do in that accept is generally expected/understood to return void, and I prefer to avoid a potential confusion about this being what it is not (a java.util.function Consumer type interface; there is no ObtToLongFunction in java.util.function so....